components/Tweet/index.jsx

```diff
@@ -42,24 +49,26 @@ const Tweet = () => {
     setIsSubmitting(true);
 
     try {
-      const has100kVeOlas = await checkVeolasThreshold(
-        account,
-        '100000000000000000000000',
-      );
-      if (!has100kVeOlas) {
-        throw new Error(
-          'You must hold at least 100k veOLAS to propose a tweet.',
-        );
+      const has100kVeOlas = await checkVeolasThreshold(account, HUNDRED_K_OLAS);
+      if (!isStaging && !has100kVeOlas) {
+        notifyError('You must hold at least 100k veOLAS to propose a tweet.');
+        return;
       }
 
+      const signature = await signMessageAsync({
+        message: `I am signing a message to verify that I propose a tweet starting with ${getFirstTenCharsOfTweet(
+          tweetOrThread,
+        )}...`,
```

Comment:
```suggestion
        )}…`,
```
In general, use the ellipsis character rather than 3 full stops. It's more efficient and looks better.